src/coreclr/debug/ee/debugger.cpp

```diff
@@ -10468,34 +10468,6 @@ bool Debugger::HandleIPCEvent(DebuggerIPCEvent * pEvent)
         }
         break;
 
-    case DB_IPCE_IS_OPTS_DISABLED:
```

Comment:
You can also remove all the definitions of `DB_IPCE_IS_OPTS_DISABLED` and `DB_IPCE_IS_OPTS_DISABLED_RESULT` and also `IsOptsDisabledData`